### PR TITLE
426 new case studies

### LINKED ISSUE #426: new case studies

### PR DESCRIPTION
**Pull request by Arfah**
#closes #426

### CODE REVIEW

**Comment by tansaku** [comment]
@Arfah - do you need proof-reading on this?  I assume you've tested the change on your local computer so that everything looks okay?  

Apologies for extra questions - just getting used to flow here

**Comment by Arfah** [comment]
everything looks okay but yes if someone can cast one final eye over it
then that would be good. However I've edited it to the best of my ability.

On Tue, Oct 20, 2015 at 9:50 AM, Sam Joseph <notifications@github.com>
wrote:

> @Arfah <https://github.com/Arfah> - do you need proof-reading on this? I
> assume you've tested the change on your local computer so that everything
> looks okay?
>
> Apologies for extra questions - just getting used to flow here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/makersacademy/website/pull/427#issuecomment-149481219>
> .
>



-- 
Arfah Farooq
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs3LjvP2W64Jb8q4Y9g2PW5vMrTT56dFrPf3gdQKj02?t=http%3A%2F%2Fwww.arfahfarooq.com%2F&si=5605859451731968&pi=ccd2e34d-5c25-4fb5-c42f-e30df6af2972>
Say hello on Twitter: @Arf_22
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs3LjvP2W64Jb8q4Y9g2PW5vMrTT56dFrPf3gdQKj02?t=http%3A%2F%2Fwww.twitter.com%2Farf_22&si=5605859451731968&pi=ccd2e34d-5c25-4fb5-c42f-e30df6af2972>
 and Linkedin
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs3LjvP2W64Jb8q4Y9g2PW5vMrTT56dFrPf3gdQKj02?t=http%3A%2F%2Fuk.linkedin.com%2Fpub%2Farfah-farooq%2F44%2F600%2F602%2F&si=5605859451731968&pi=ccd2e34d-5c25-4fb5-c42f-e30df6af2972>

File: data/graduates.yml [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+  cohort_month: August
+  cohort_year: 2015
+  categories:
+    - career changer
+    - female
+  case_study:
+    - question: What were you doing before joining Makers Academy?
+      answer: I was working as a data journalist at the Sunday Times.
+    - question: What made you want to start learning to code in the first place?
+      answer: I had been doing some programming for work and I wanted to do it properly and full time.
```
Comment (pitchinvasion) [inline data/graduates.yml]: I'd change that just "wanted to do it full-time"

**Comment by pitchinvasion** [comment]
:shipit: made some proofreading descriptions

File: data/graduates.yml [hunk 2]
```diff
@@ -1,9 +1,71 @@
+- first_name: Nicola
+  last_name: Hughes
+  image: nicola.jpg
+  summary: Data Journalist to Junior Developer at ThoughtWorks
+  testimonial: I can code and I got a job and I'm now looking forward to working in the tech industry and not just media.
+  cohort_month: August
```
Comment (nikeshashar) [inline data/graduates.yml]: Wrong cohort

File: data/graduates.yml [hunk 3]
```diff
@@ -1,9 +1,71 @@
...
+      answer: My biggest worry was not knowing what I wanted to do with the skills I gained and ThoughtWorks ticks every single box in terms of what I want right now so that worry is now gone.
+    - question: Sum up your Makers Academy experience
+      answer: Be brave and go for it. You will get a lot of support from everyone.
+
+- first_name: Kirsten
+  last_name: Jones
+  image: kirsten.jpg
+  summary: Lawyer to Junior Developer at Financial Times
+  testimonial: Makers Academy is an incredible environment to learn in. You don't just learn to code here but you also have lots of fun and meet great people and have an amazing support network of expert coders and wellbeing support.
+  cohort_month: August
```
Comment (nikeshashar) [inline data/graduates.yml]: Wrong cohort

File: data/graduates.yml [hunk 4]
```diff
@@ -1,9 +1,71 @@
...
+    - question: Sum up your Makers Academy experience
+      answer: It's intense, challenging but amazing.
+
 - first_name: Zeina
   last_name: Dajani
   image: Zeina.jpg
   summary: Student to Junior Developer at FNZ
   testimonial: Makers Academy changes your life, how you look at it and what you want to do with it.
-  cohort_month: June
+  cohort_month: August
```
Comment (nikeshashar) [inline data/graduates.yml]: Wrong cohort

File: data/graduates.yml [hunk 5]
```diff
@@ -60,6 +122,62 @@
...
+      answer: I wasn't really worried before joining Makers. I thought I'd ace it. The worries came after the first week and I couldn't make my code work. But now I know I can just step through it, read a lot, ask good questions, and it will work eventually.
+    - question: Sum up your Makers Academy experience
+      answer: A total emotional rollercoaster with great people on the ride with me!
+
+- first_name: Retesh
+  last_name: Bajaj
+  image: retesh.jpg
+  summary: Cardiology Doctor who learnt to code
+  testimonial: It was memorable and life changing and my life won't be the same again.
+  cohort_month: September
```
Comment (nikeshashar) [inline data/graduates.yml]: July

File: data/graduates.yml [hunk 6]
```diff
@@ -60,6 +122,62 @@
     - question: Sum up your Makers Academy experience
       answer: Coding and great friends.
 
+- first_name: Sara
+  last_name: O'Connor
+  image: sara.jpg
+  summary: Childrens Author to Junior Developer at Exceedra
+  testimonial: A total emotional rollercoaster with great people on the ride with me!
+  cohort_month: June
```
Comment (nikeshashar) [inline data/graduates.yml]: March

**Comment by Arfah** [comment]
@nikeshashar  you are wrong. The cohort actually refers to month they graduated. See website.

**Comment by nikeshashar** [comment]
@Arfah 

image

Zeina didn't graduate in June 


image

Alex didn't graduate in April

**Comment by nikeshashar** [comment]
image

**Comment by Arfah** [comment]
@nikeshashar I updated it already it's in the latest commit. Anyway I'm merging.

**Pull request merged**